__tests__/mockClient.test.js

```diff
@@ -0,0 +1,96 @@
+const Q = require('q');
+const request = Q.denodeify(require('request'));
+const MockClient = require('../src/client/Client');
+const MOCKSERVER_URI = 'localhost:3000';
+
+describe('when adding routes on the same uri', () => {
+	let mockClient;
+	beforeEach(() => {
```

Comment:
Would you like to add a blank line above `beforeEach` to separate between the declaration statement an the functions? ;)